R/replicates.R

```diff
@@ -0,0 +1,66 @@
+# replicates.R
```

Comment:
Consider moving as a function within the `collapse.R` file

Reply:
Will leave as-is -- there might be more content related to replicates released in future versions.